students/BrianNisonger/Lesson4/basic_operations.py

```diff
@@ -0,0 +1,52 @@
+from peewee import *
+from customer_model import Customer
+from customer_model import database
+
+
+def add_customer(**kwargs):
+    try:
+        kwargs['credit_limit'] = float(kwargs['credit_limit'])
+    except ValueError as err:
+        raise
```

Comment:
You should log the error (err) you are saving. :) Otherwise there is no point in catching it.